proto/proto/requests.proto

```diff
@@ -1,13 +1,15 @@
 syntax = "proto3";
 package requests;
 
-import "account_id.proto";
+import "account.proto";
 import "block_header.proto";
 import "digest.proto";
 import "note.proto";
 
+message EmptyRequest {}
```

Comment:
Is this the best practice for handling empty messages in protobuf?

Reply:
I believe that this is a standard way to represent empty messages. To have a generic `message EmptyRequest` and ` message EmptyResponse` to clarify the functions that don't take arguments or return data.

The other solution would be to create specific messages like: 
`message ListNullifiersRequest` and `message ListNullifiersResponse` ( do this for the 3 endpoints ) even though all the request messages will be empty.

Reply:
If we want to keep the `message EmptyRequest` and `message EmptyResponse` scheme I believe that we should also add it to other endpoints to standardise the interface. 

@plafer what do you think?

Reply:
From this [SO thread](https://stackoverflow.com/q/50993815/3499862), we should use [`google.protobuf.Empty`](https://protobuf.dev/reference/protobuf/google.protobuf/#empty), unless we think we'll someday want to add arguments (without breaking change). So from what I understand,

1. Define a specific message type for each `rpc` that we think we might want to add fields to at some point in the future
2. Otherwise, use `google.protobuf.Empty`

Here, it seems possible to me that we'd want to support optional parameters, so I would define 3 different request types.

Reply:
> Here, it seems possible to me that we'd want to support optional parameters, so I would define 3 different request types.

Yep - let's go with this.